project-service/app.js

```diff
@@ -0,0 +1,27 @@
+var express = require("express");
```

Comment:
Lets move out from **js** and move to **ts** where you can design program in OOP way. Use class based structure